controllers/hetznerbaremetalhost_controller_test.go

```diff
@@ -44,8 +44,7 @@ import (
 )
 
 const (
-	bmMachineName = "bm-machine-host-testing"
-	hostName      = "test-host"
+	hostName = "test-host"
```

Comment:
can you move this code change to a separate PR? This has no relation to the PR title.